core/src/main/java/net/siegerpg/siege/core/listeners/MountSteer.java

```diff
@@ -165,14 +170,12 @@ public void useMobEgg(PlayerInteractEvent e) {
 
 			//cooldown
 			if (currentCooldown.containsKey(player)) {
```

Comment:
So here's where things get a little difficult. I'll do a step by step logic of everything up to this boolean:
1. Player uses the spawn egg
2. Player gets added to the cooldown
3. The setCancelled method will cancel the firing of the event
4. If the player is already on the mount (cachedMounts contains the player) then they get off the mount END
5. If player is sneaking then method ends **This needs code from line 157-160**
6. If the cooldown contains the player ->
7. If the cooldown has been more than 20 seconds remove player from cooldown
8. If not, then end method and send message

Let's take a scenario
A. I spawn my mount in for the first time. So I get added to the cooldown, and then later in the method I get a message saying my mount is still on cooldown because it hasn't been more than 2 seconds later.

B. I spam click my mount and I get what has happened above, but the second time/third time I will get a my mount is still on cooldown. (So that's good)